package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@react-three/fiber": "^8.17.10",
-    "@tabler/icons-react": "^3.21.0",
```

Comment:
please make sure tabler/icons-react is updated, using the previous version might end up breaking in the production. Run pnpm build to make sure, if it doesnt work, you can still make the pull request, just remove the component from the page.tsx file.